x-pack/plugins/infra/public/components/alerting/inventory/validation.tsx

```diff
@@ -6,8 +6,6 @@
 
 import { i18n } from '@kbn/i18n';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { isNumber } from 'lodash';
```

Comment:
I'm not sure it works with our version of lodash, but did you consider/try writing this as `import isNumber from 'lodash/isNumber'`?

That way you tell webpack to not bundle the whole thing, and we remove some duplication existing in `metric_threshold/components/validation.tsx`

(This could be automated with [babel-plugin-lodash](https://www.npmjs.com/package/babel-plugin-lodash), but that's outside the scope of this PR)

Reply:
no, I wasn't aware that our custom @elastic/lodash version supported that. I'll try that out.

Reply:
The old lodash types don't support that :disappointed: Do you think the duplication of this one-liner is problematic?

Reply:
No, not at all